src/nft/components/collection/CollectionAsset.test.tsx

```diff
@@ -0,0 +1,54 @@
+import { NftStandard } from 'graphql/data/__generated__/types-and-hooks'
+import { GenieAsset, Markets, UniformAspectRatios } from 'nft/types'
+import { render } from 'test-utils/render'
+
+import { CollectionAsset } from './CollectionAsset'
+
+const AssetFixture: GenieAsset = {
```

Comment:
can you add this re-useable asset to test utils? I imagine several other nft tests will want to use a mocked not for sale asset

Reply:
good point